pkg/bot/slack.go

```diff
@@ -220,3 +262,64 @@ func (sm *slackMessage) Send() error {
 
 	return nil
 }
+
+// SendEvent sends event notification to slack
+func (b *Slack) SendEvent(ctx context.Context, event events.Event) error {
+	if !b.notify {
+		b.log.Info("Notifications are disabled. Skipping event...")
+		return nil
+	}
+
+	b.log.Debugf(">> Sending to slack: %+v", event)
+	attachment := b.formatMessage(event, b.Notification)
+
+	targetChannel := event.Channel
+	if targetChannel == "" {
+		// empty value in event.channel sends notifications to default channel.
+		targetChannel = b.ChannelName
+	}
+	isDefaultChannel := targetChannel == b.ChannelName
+
+	channelID, timestamp, err := b.Client.PostMessage(targetChannel, slack.MsgOptionAttachments(attachment), slack.MsgOptionAsUser(true))
+	if err != nil {
+		postMessageWrappedErr := fmt.Errorf("while posting message to channel %q: %w", targetChannel, err)
+
+		if isDefaultChannel || err.Error() != channelNotFoundCode {
+			return postMessageWrappedErr
+		}
+
+		// channel not found, fallback to default channel
+
+		// send error message to default channel
+		msg := fmt.Sprintf(sendFailureMessageFmt, targetChannel, err.Error(), targetChannel)
+		sendMessageErr := b.SendMessage(ctx, msg)
+		if sendMessageErr != nil {
+			return multierror.Append(postMessageWrappedErr, sendMessageErr)
+		}
+
+		// sending missed event to default channel
+		// reset event.ChannelName and send event
```

Comment:
still not fixed, https://github.com/kubeshop/botkube/pull/662#discussion_r931988041

```suggestion
		// reset channel, so it will be defaulted
```

Reply:
I fixed that, but in line 376 :( I didn't see the other occurrence. Sorry!